doc/news/changes.h

```diff
@@ -201,6 +201,14 @@ inconvenience this causes.
 <h3>Specific improvements</h3>
 
 <ol>
+ <li> New: Added an optional string parameter to the ParameterHandler::read_input ()
```

Comment:
@bangerth is it ok like that or would you prefer to have arguments and two separate `read_input()` functions? I am not sure if doxygen will pick it up from `changes.h`.